api/include/opentelemetry/trace/span_context.h

```diff
@@ -33,6 +33,10 @@ namespace trace_api = opentelemetry::trace;
 class SpanContext final
 {
 public:
+  // An invalid SpanContext.
```

Comment:
Where do we require this?

Reply:
we need for the `StartSpan` function in `etw_tracer_exporter.h`, line 115
[link](https://github.com/open-telemetry/opentelemetry-cpp/blob/4df4ce60ff3e274e9959b1ae118ca999f3b8a25b/exporters/etw/include/opentelemetry/exporters/etw/etw_tracer_exporter.h#L115)

Reply:
Please double-check how it's done in the `ostream` exporter. Why didn't they need it?

Reply:
Have replaced with `trace::SpanContext::GetInvalid()`, an existing method.